coap-gateway/service/signOff_test.go

```diff
@@ -15,16 +15,16 @@ func TestSignOffHandler(t *testing.T) {
 
 	co := testCoapDial(t, testCfg.GW_HOST)
 	require.NotNil(t, co)
-	signUpResp := testSignUp(t, CertIdentity, co)
+	_ = testSignUp(t, CertIdentity, co)
 	err := co.Close()
 	require.NoError(t, err)
 
 	tbl := []testEl{
 		{"Bad query", input{coapCodes.DELETE, `{}`, []string{"di=%"}}, output{coapCodes.BadOption, "invalid URL escape", nil}, true},
 		{"Bad request (no userId)", input{coapCodes.DELETE, `{}`, []string{"di=" + CertIdentity}}, output{coapCodes.BadRequest, "invalid user id", nil}, true},
-		{"Bad request (invalid userId)", input{coapCodes.DELETE, `{}`, []string{"di=" + CertIdentity, "accesstoken=" + signUpResp.AccessToken, "uid=0"}}, output{coapCodes.InternalServerError, "invalid ownerClaim", nil}, true},
 		{"Bad request (missing access token)", input{coapCodes.DELETE, `{}`, []string{"di=" + CertIdentity, "uid=0"}}, output{coapCodes.BadRequest, `invalid access token`, nil}, true},
-		{"Deleted0", input{coapCodes.DELETE, `{}`, []string{"di=" + CertIdentity, "accesstoken=" + signUpResp.AccessToken, "uid=" + signUpResp.UserID}}, output{coapCodes.Deleted, nil, nil}, false},
+		// not supported because JWT has length more than 255 bytes
```

Comment:
This shouldn't be problem in iotivity plgd branch, or is there some other layer that fails with token longer than 255b?

Reply:
this test doesn't use iotivity-lite device - it uses go-coap/tcp/client which emulates device